src/Microsoft.Health.Fhir.Core/Features/Operations/Export/Models/ExportJobRecord.cs

```diff
@@ -26,7 +26,6 @@ public ExportJobRecord(Uri exportRequestUri)
             Status = OperationStatus.Queued;
             Id = Guid.NewGuid().ToString();
             QueuedTime = DateTimeOffset.Now;
```

Comment:
I think we should normalize to UTC.

Reply:
I will update it. This logic will change a little bit as well since I am planning to make a stored procedure for inserting the job record. 

The reason I am thinking of using stored procedure is because we want to check the job hash first to see if there is any existing active job with matching hash and return that instead of creating a new record.